UnityProject/Assets/Scripts/Systems/Explosions/NodeTypes/ExplosionNode.cs

```diff
@@ -127,6 +110,33 @@ public virtual float DoDamage(Matrix matrix, float DamageDealt, Vector3Int v3int
 			return EnergyExpended;
 		}
 
+		protected void DamageLayers(float DamageDealt, Vector3Int v3int)
```

Comment:
```suggestion
		protected void DamageLayers(float damageDealt, Vector3Int position)
```
You are turning into Bod

Reply:
I just copy pasted those tbh

Reply:
oh no the Fox curse : P